hack/test-cluster-class.sh

```diff
@@ -0,0 +1,12 @@
+#/bin/bash
+
```

Comment:
Way too complicated. Use the existing tooling for env replacement. We don't want to do that manually.
Write out both files with `clusterctl generate yaml` instead

Add the missing env variables to the project [example envfile](https://github.com/ionos-cloud/cluster-api-provider-ionoscloud/blob/main/envfile.example)

Please rewrite the bash script to something like this:

```bash
source .envfile
export CLUSTER_CLASS_NAME="xyz"
export NAMESPACE="default"

clusterctl generate yaml --from templates/clusterclass-template.yaml > ...
clusterctl generate yaml --from templates/cluster-template-topology > ...

# validate
clusterctl alpha topology plan -f ... -f.. -o ...

# apply
# kubectl apply -f generated-yaml-clusterclass
# kubectl apply -f generated-yaml-cluster-template
```

The idea is to have a script which will help me understand what I have to do. From what I understand currently. This is what I would have to do in order to make things run.

Reply:
better?